### PR TITLE
Draw vulkan event linking arrow based on slices' distance.

### PR DESCRIPTION
**Pull request by stellama0208**
- Extend the arrow to the start of the linked gpu queue slices group.
 - Bug: http://b/148964552.

### CODE REVIEW

File: gapic/src/main/com/google/gapid/perfetto/models/VulkanEventTrack.java [hunk 1]
```diff
@@ -51,6 +51,14 @@
   private static final String SLICES_SQL =
       "select " + BASE_COLUMNS + " from %s " +
           "where ts >= %d - dur and ts <= %d order by ts";
+  private static final String QUEUE_GROUP_START_SQL =
+      "select submission_id, min(ts) start from gpu_track t left join gpu_slice s " +
+          "on (t.id = s.track_id) where t.name like \'queue%%\' group by submission_id";
```
Comment (chrisforbes) [inline gapic/src/main/com/google/gapid/perfetto/models/VulkanEventTrack.java]: This should probably find the correct tracks by scope rather than by name?

Comment (rschiu) [reply gapic/src/main/com/google/gapid/perfetto/models/VulkanEventTrack.java]: Yes, please never rely on the name of the track (or almost anything for that matter) as an identifier.

File: gapic/src/main/com/google/gapid/perfetto/models/VulkanEventTrack.java [hunk 2]
```diff
@@ -95,13 +104,14 @@ public VulkanEventTrack(QueryEngine qe, GpuInfo.VkApiEvent vkApiEvent) {
             data.commandBuffers[i] = row.getLong(5);
             data.submissionIds[i] = row.getLong(6);
             data.args[i] = args.getOrDefault(row.getLong(7), ArgSet.EMPTY);
+            data.dists[i] = row.getLong(8);
```
Comment (rschiu) [inline gapic/src/main/com/google/gapid/perfetto/models/VulkanEventTrack.java]: Maybe a comment explaining what dists is.

File: gapic/src/main/com/google/gapid/perfetto/models/VulkanEventTrack.java (hunk 1 above)
Comment (stellama0208) [reply gapic/src/main/com/google/gapid/perfetto/models/VulkanEventTrack.java]: Thanks for pointing this out!

File: gapic/src/main/com/google/gapid/perfetto/models/VulkanEventTrack.java (hunk 2 above)
Comment (stellama0208) [reply gapic/src/main/com/google/gapid/perfetto/models/VulkanEventTrack.java]: Thanks! Done in a new commit.

**Pull request merged**